src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -316,4 +318,32 @@ export class ConsultationQuestionnaireComponent implements OnInit, AfterViewInit
     });
   }
 
+  scrollToFirstInvalidControl() {
```

Comment:
@Nishanthh Bro a similar function is already defined in modular.functions.ts. Can that function itself be reused here?

Reply:
@avikt yes, used the modular function now